llvm/utils/TableGen/DXILEmitter.cpp

```diff
@@ -653,12 +641,18 @@ static void EmitDXILOperation(RecordKeeper &Records, raw_ostream &OS) {
   for (auto *Record : OpIntrProps) {
     DXILOps.emplace_back(DXILOperationDesc(Record));
   }
+  // Sort by opcode.
```

Comment:
Wondering if there's anything in place to catch duplicate entries?  Previously I think these would have been quietly swallowed? Now we'd probably get an error in the switch statement.  It'd probably be nicer to catch these while running tablegen rather than later?

Reply:
Seems like a good idea, I put a check in just after the sort.